src/lib/styles/forms.less

```diff
@@ -1,50 +1,136 @@
-/*************************
- * From publish reset.css:
- */
-button, input, select, textarea {
-  margin: 0;
-}
-
-input, select {
-  vertical-align: middle;
+form {
+  display: -webkit-box;
+  display: -moz-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: horizontal;
+  -moz-box-orient: horizontal;
+  -o-box-orient: horizontal;
+  -webkit-box-lines: multiple;
+  -moz-box-lines: multiple;
+  -o-box-lines: multiple;
+  -webkit-flex-flow: row wrap;
+  -ms-flex-flow: row wrap;
+  flex-flow: row wrap;
+  -webkit-box-pack: start;
+  -moz-box-pack: start;
+  -o-box-pack: start;
+  -ms-flex-pack: start;
+  -webkit-justify-content: flex-start;
+  justify-content: flex-start;
+  -webkit-box-align: baseline;
+  -moz-box-align: baseline;
+  -o-box-align: baseline;
+  -ms-flex-align: baseline;
+  -webkit-align-items: baseline;
+  align-items: baseline;
+  -ms-flex-line-pack: start;
+  -webkit-align-content: flex-start;
+  align-content: flex-start;
 }
 
-/* candidate for removal? */
 label {
-  margin: 0;
-  padding: 0;
+  -webkit-box-flex: 1;
+  -moz-box-flex: 1;
+  -o-box-flex: 1;
+  box-flex: 1;
+  -webkit-flex: 0 1 100%;
+  -ms-flex: 0 1 100%;
+  flex: 0 1 100%;
+  flex-basis: 100%;
+  -ms-flex-preferred-size: 100%;
+  -webkit-flex-basis: 100%;
   border: 0;
+  padding: 0;
+  margin: 0 0 3px 0;
   font-size: 100%;
   font-weight: 400;
   vertical-align: baseline;
   background: 0 0;
+  outline: none;
+  cursor: pointer;
 }
 
-input[type=radio] {
-  vertical-align: text-bottom;
+label + textarea {
+  vertical-align: top;
```

Comment:
This is a new rule, but we aren't using textarea anywhere. It just looks better with the horizontal form layout.